src/util.ts

```diff
@@ -32,3 +31,8 @@ export const CONFIG_PATHS = Object.fromEntries(
     return [name, path];
   }),
 ) as Record<typeof CONFIG_FOLDER_NAMES[number], string>;
+
+const QUICK_CSS_FILE = join(CONFIG_PATHS.quickcss, "main.css");
+if (!existsSync(QUICK_CSS_FILE)) {
+  writeFileSync(QUICK_CSS_FILE, "");
+}
```

Comment:
Shouldn't we create the files for all? I know this pr is for quick css but still :shrug:

Reply:
This is the only file we needed to make. No files to create for plugins and themes and we don't know the file names for settings (plus they'll be created when they're needed).